src/schema/response.rs

```diff
@@ -1,9 +1,15 @@
 use crate::schema::{Assertion, Issuer, Status};
 use crate::signature::Signature;
 use chrono::prelude::*;
-use serde::Deserialize;
+use serde::{Deserialize};
 use snafu::Snafu;
 use std::str::FromStr;
+use quick_xml::events::{BytesEnd, BytesStart, Event, BytesDecl};
+use quick_xml::Writer;
+use std::io::Cursor;
+
+const NAME: &str = "saml2p:Response";
+const SCHEMA:(&str, &str) = ("xmlns:saml2p", "urn:oasis:names:tc:SAML:2.0:protocol");
```

Comment:
This is a bit inconsistent with the above "vector of tuples" approach. After this is merged I'll go through an make all the structs consistent